### PR TITLE
Bundling

### PR DESCRIPTION
**Pull request by rossjrw**
As the size of the interwiki has grown to multiple JS files, it is no longer efficient to keep them entirely uncompiled and separate from each other. I'd like to introduce bundling to merge all JS files together as a compiled step.

This will make deployment easier, will make testing easier (and it may even be possible to test using a dev server), and could even be extended to give branches the option of changing the JS file location without needing to fork the project and change it in multiple places.

### CODE REVIEW

File: html/interwikiFrame.html [hunk 1]
```diff
@@ -1,70 +1,19 @@
...
+      <p></p>
     </div>
-    <div id="resizer-container"></div>
-  </body>
-</html>
+  </div>
+  <div id="resizer-container"></div>
+</body>
+
+</html>
```
Comment (emmiegit) [inline html/interwikiFrame.html]: Missing newline at end of file

**Review by emmiegit — APPROVED** [review]

**Comment by emmiegit** [comment]
Question: The use of `var` etc everywhere is to support very old browsers without the use of Babel or something, right? So no lambdas or classes either.

File: html/interwikiFrame.html (hunk 1 above)
Comment (rossjrw) [reply html/interwikiFrame.html]:
```suggestion
</html>

```

**Comment by rossjrw** [comment]
> Question: The use of `var` etc everywhere is to support very old browsers without the use of Babel or something, right? So no lambdas or classes either.

That's correct. Now that we have a compile step, there's basically no benefit to doing this - it was to save the effort of setting up Babel, which would now be fairly trivial. The only reasons that I've kept the ES5 restriction are that it'd be a bit of work to update everything, and I'd like to be able to easily merge this into #8. ES5 doesn't have the `class` keyword, but has classes conceptually, and it doesn't have arrow function syntax `() => { ... }` but does have anonymous functions (which is what I infer from 'lambda').

The biggest thing ES5 is missing IMO is Promises, so I have to use this disgusting callback structure everywhere

**Pull request merged**